Gemfile

```diff
@@ -145,7 +145,6 @@ gem 'mime-types', '< 3.0.0', require: false
 
 # Assets
 gem 'bootstrap-sass', '~> 2.3.2.2'
-gem 'sass-rails', '~> 5.0.7'
```

Comment:
I'm not _entirely_ sure about this.

The [compass-rails install notes](https://github.com/Compass/compass-rails#installation) say to add both.

Incidentally, compass-rails is deprecated, so I'm not sure we should rely on it doing the right thing when it comes to upgrades etc.

> Don't start new projects with compass, it is no longer supported, see the compass statement
>
> https://github.com/Compass/compass-rails#compass-rails